### DISCUSSION TITLE
When will current master be be pushed as a package?

### DISCUSSION THREAD

**Discussion by hcr-mit** [espressif/arduino-esp32#9931] [Q&A] [answered]
Hi, 
   I am working on getting an ESP32S3 board setup to act as a USB device. The board has an SD card reader on a SPI. I am in need of the new SD_MMC and support code to make this work.  
   If I can get an idea of when the current code will be packaged, I can better estimate when this R&D can be delivered. TIA

**Comment by me-no-dev** [comment, marked as answer]
it has just been released as 3.0.2

**Reply by hcr-mit** [reply]
Thank you
